aspnetapp/aspnetapp/Dockerfile

```diff
@@ -0,0 +1,3 @@
+﻿FROM microsoft/aspnet
```

Comment:
The same comments I made to the [Docker sample ](https://github.com/kendrahavens/dockerize-dotnet-app/issues/1#issuecomment-273883533) apply here as well.